src/index.html

```diff
@@ -11,7 +11,184 @@
+            >
+          </label>
+          <label>Password:
+            <input 
+            type="password" 
+            id="pwd" 
+            name="pwd" 
+            required 
+            minlength="8" 
+            maxlength="16"
+            >
+          </label>
+        </fieldset>
+      </div>
+    
+      <div class="group">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <label class="label-margin">
+            Do you love cats?
+            <label>
+              <input 
+              type="radio" 
+              id="cats-yes" 
+              name="cats" 
+              value="yes" 
+              required
+              >
+              Yes
+            </label>
+          
+            <label>
+              <input 
+              type="radio" 
+              id="cats-no" 
+              name="cats" 
+              value="no" 
+              required
+              >
+            No
+            </label>
+          </label>
+          
+          <label for="color" class="label-margin">
+            What is your favorite color?
+            <input 
+            type="color" 
+            id="color" 
+            name="color" 
+            required
+            >
+          </label>
+          <label for="bedTime" class="label-margin">
+            What time do you go to bed?
+            <input 
+                type="time" 
+                id="bedTime" 
+                name="bedTime" 
+                required
+                step="1"
+            >
+          </label>
+          <label class="label-margin">
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+          <label>
+            How do you rate our work?
+            <input 
+            type="range" 
+            min="0" 
+            max="10" 
+            step="1" 
+            name="rating" 
+            id="rating"
+            value="0"
+            >
+          </label>
+        </fieldset>
+      </div>
+  
+      <div class="group">
+        <fieldset>
+          <legend>Additional info:</legend>
+          <label>
+            Comments:
+            <textarea 
+              id="comments" 
+              name="comments" 
+              rows="2" 
+              cols="20"
+            >
+            </textarea>
+          </label>
+          <br>
+          <label for="recommend-yes">Would you recommend us?</label>
+          <select id="recommend" name="recommend">
+            <option id="recommend-yes" value="yes">Yes</option>
+            <option value="no">No</option>
+          </select>
+        </fieldset>
+      </div>
+
+      <label for="submitButton"></label>
+      <input 
+      type="submit" 
+      id="submitButton" 
+      value="Submit"
+      > 
```

Comment:
please be attentive with spaces of attributes
```suggestion
      <input 
        type="submit" 
        id="submitButton" 
        value="Submit"
      > 
```